cmem/encoder_test.go

```diff
@@ -6,51 +6,50 @@ package cmem
 
 import (
 	"bytes"
-	"encoding/binary"
-	"strconv"
 	"testing"
+	"encoding/binary"
 )
 
 func TestEncode(t *testing.T) {
-	for i, tc := range []struct {
```

Comment:
Why was this deleted?

Reply:
The original test case was not correct for variable length data, it's supposed to handle it same as fix length array. So I removed the second test data, and the loop is not needed anymore